flink-autoscaler/pom.xml

```diff
@@ -107,6 +107,12 @@ under the License.
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>${awaitility.version}</version>
```

Comment:
Should this be test scoped?

Reply:
Yeah it should be.

Reply:
resolved with 5256ad3090e996e92d12e653609f1e4595ab5fc1